src/test-apps/Makefile.am

```diff
@@ -74,7 +73,7 @@ SUBDIRS                                               = \
 # Define a package-specific test driver (for 'make check').
 #
 
-LOG_DRIVER=$(abs_srcdir)/weave-parallel-test-engine
+LOG_DRIVER=$(abs_top_nlbuild_autotools_dir)/third_party/autoconf/weave-parallel-test-engine
```

Comment:
It may be OK for openweave-core to place this here in its own copy of nlbuild-autotools; however, I don't think this belongs in nlbuild-autotools unless the changes can be generically upstreamed.

Reply:
I wholeheartedly agree.  Now, there is a set of changes that I could see being generically useful:

* idea of grace runs for flaky tests (though it pains me to admit it, it seems like a pragmatic feature)
* better signal handling (e.g. if the test under consideration was killed via a signal)
* an environment variable that conveys a unique number to the program under test in a parallel execution scenario